solr/core/src/test/org/apache/solr/update/SolrIndexConfigTest.java

```diff
@@ -208,4 +210,16 @@ public void testToMap() throws Exception {
 
     assertEquals(mSizeExpected, m.size());
   }
+  
+  public void testMaxCommitMergeWaitSeconds() throws Exception {
+    SolrConfig sc = new SolrConfig(TEST_PATH().resolve("collection1"), "solrconfig-test-misc.xml");
+    assertEquals(-1, sc.indexConfig.maxCommitMergeWaitMillis);
+    assertEquals(IndexWriterConfig.DEFAULT_MAX_COMMIT_MERGE_WAIT_MILLIS, sc.indexConfig.toIndexWriterConfig(h.getCore()).getMaxCommitMergeWaitMillis());
+    System.setProperty("solr.tests.maxCommitMergeWait", "10");
+    sc = new SolrConfig(TEST_PATH().resolve("collection1"), "solrconfig-test-misc.xml");
+    assertEquals(10, sc.indexConfig.maxCommitMergeWaitMillis);
+    assertEquals(10, sc.indexConfig.toIndexWriterConfig(h.getCore()).getMaxCommitMergeWaitMillis());
+    System.clearProperty("solr.tests.maxCommitMergeWait");
```

Comment:
I don't think we need to clear this, the test runner takes care of it and will reset it to whatever it was previously if it had been set?

Reply:
Please verify if this is so before removing.  Perhaps there may be a difference in behavior between gradle & IntelliJ; so try both.  I thought clearing wasn't necessary as well but I recall hearing from someone (@dweiss ?) that the auto-clearing wasn't working.

Reply:
Yes, I checked. I believe the cleanup is after the class. I discussed with @madrob, and I'll move the cleanup to a `tearDown()` method